cmake/define_libwiredtiger.cmake

```diff
@@ -0,0 +1,103 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+#  See the file LICENSE for redistribution information
+#
+
+# define_wiredtiger_library(target type)
+# A helper that defines a wiredtiger library target. This defining a set of common targets and properties we
+# want to associated to any given 'libwiredtiger' target. Having this as a macro allows us to de-duplicate common
```

Comment:
Seems like there is a typo here
"want to *be* associated to"
"want to associate to"

Reply:
Yep, true. I fixed up the wording.